jsen/jsen.d.ts

```diff
@@ -0,0 +1,45 @@
+// Type definitions for jsen (JSON Sentinel)
+// Project: https://github.com/bugventure/jsen
+// Definitions by: Vladimir Đokić <https://github.com/vladeck/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+interface IJsenFormats {
```

Comment:
http://www.typescriptlang.org/docs/handbook/writing-definition-files.html
> Naming Conventions
In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.